javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+const books = [
+  {
+    isbn: '123143234143488',
+    title: 'Loop de Loop',
+    author: 'Mr. Looper'
+  },
+  {
+    isbn: '59286255023',
+    title: 'Straight Lines Only',
+    author: 'Curvy Curvester'
+  },
+  {
+    isbn: '039443344526',
+    title: 'Big Balloons',
+    author: 'Flyn Flyer'
+  }
+];
+
+console.log('books array: ', books);
+console.log('typeof books: ', typeof books);
+
+const jsonBooks = JSON.stringify(books);
+console.log('json books: ', jsonBooks);
+
+const jsonStudent = `{
+  "Number id": 235024,
+  "name": "Greg"
+}`;
```

Comment:
This is fine, but this only works because you're using a template literal. If you tried to use a single quote, this wouldn't be correct. I'll approve it, but I want to you understand that a JSON string must be wrapped with single quotes on that outside.